overlord/hookstate/hookmgr.go

```diff
@@ -143,14 +143,37 @@ func (m *HookManager) Stop() {
 	m.runner.Stop()
 }
 
+func (m *HookManager) snapContext(contextID string) (context *Context, err error) {
+	var contexts map[string]string
+	err = m.state.Get("snap-contexts", &contexts)
```

Comment:
That's probably "snap-cookies" if we end up moving ahead with that change, which does sound much nicer.